tests/fuzzer/fuzzer.py

```diff
@@ -181,16 +206,11 @@ def test_stdcm(
     stdcm_payload = make_stdcm_payload(scenario, path, rolling_stock)
     r = post_with_timeout(editoast_url + "stdcm/", json=stdcm_payload)
     if r.status_code // 100 != 2:
-        if r.status_code // 100 == 4 and "no_path_found" in r.content.decode("utf-8"):
+        content = r.content.decode("utf-8")
+        if r.status_code // 100 == 4 and ("no_path_found" in content or "No path could be found" in content):
```

Comment:
```suggestion
        if r.status_code // 100 == 4 and "No path could be found" in content:
```
The detailed error message should remain there even when the error code will be fixed